steps/utility/utility.go

```diff
@@ -52,6 +54,17 @@ var _ = gauge.Step("Validate route url for pipelines tutorial", func() {
 	}
 })
 
+var _ = gauge.Step("Validate deployed application", func() {
```

Comment:
1. Why didn't you reuse existing step?
2. You only dump the page content and don't valide it

Reply:
Previously I implemented that where it will validate the expected outcome but the application has different output every time and also the output is large, so thats the reason I took this approach @ppitonak

Reply:
Yes, the output is large and kind of random but you can still find a 2-3 words that never change and only validate those. 
We should definitely have only one step for two tests doing the same thing

Reply:
Done